src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -2513,8 +2512,14 @@ import ageDisplay, { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
 
   InCHlib.prototype._draw_download_menu = function () {
     const self = this;
-    const overlay = self._draw_overlay(true);
+    const width = self.stage.width();
+    const height = self.stage.height();
     const { handleFileDownloads = () => {} } = self.extHandlers;
+    const loading_div = $(`<div class="inchlib-loadingOverlay" style="width: ${width}px; height: ${height}px;"></div>`)
```

Comment:
Creates a loading overlay. This could be abstracted from the download menu, to be reused for other tasks.